ledger-core/virtual/integration/method_test.go

```diff
@@ -174,16 +175,49 @@ func TestVirtual_Method_WithExecutor(t *testing.T) {
 func TestVirtual_Method_WithExecutor_ObjectIsNotExist(t *testing.T) {
 	t.Log("C4974")
 
+	mc := minimock.NewController(t)
+
 	server, ctx := utils.NewServer(nil, t)
 	defer server.Stop()
 	server.IncrementPulse(ctx)
 
 	var (
 		objectLocal  = server.RandomLocalWithPulse()
 		objectGlobal = reference.NewSelf(objectLocal)
+		outgoing     = server.RandomLocalWithPulse()
 	)
 
-	Method_PrepareObject(ctx, server, payload.Ready, objectGlobal)
+	Method_PrepareObject(ctx, server, payload.Ready, reference.NewSelf(server.RandomLocalWithPulse()))
```

Comment:
there is no need in this prepare object (it creates random object, which is not relevant to the test)